### ISSUE TITLE
home screen: clicking users, nodes, ....blocks, go no where

### ISSUE THREAD

**Issue by europacafe** [GoodiesHQ/headscale-admin#19]
After successfully set up, the home screen where all summary of number of nodes, routes, users are shown. However, when I click those boxes, it went no where. Is it normal? I tried it on Chrome and MS Edge, same issue.

However other tabs on the left-hand side work as usual.

Did I do something wrong with my setup?

**Comment by GoodiesHQ** [comment]
Correct, the cards on the home screen displaying the values of Users/Nodes/Routes/PreAuthKeys do not actually route anywhere, mostly because only Users and Nodes have dedicated pages for them whereas the others do not.

I actually did originally have them routing to those pages, but because of the fact that not all of them actually have a visual component to them (e.g. routes and preauth keys are properties of nodes and are therefore found under their respective nodes in the nodes page), I decided against having only a subset of them be clickable and opted for having none of them clickable.
